### PR TITLE
Aurora S3 import support

### PR DESCRIPTION
**Pull request by jasonmcintosh**
Similar to the RDS S3 XtraBackup support.  Acceptance tests have passed though aurora takes forever and a day to run them.

### CODE REVIEW

**Comment by jasonmcintosh** [comment]
Let me know if I've missed anything @bflad :)  Should be nearly identical to the one for a db instance.

File: aws/resource_aws_rds_cluster.go [hunk 1]
```diff
@@ -432,6 +471,97 @@ func resourceAwsRDSClusterCreate(d *schema.ResourceData, meta interface{}) error
 
 		log.Printf("[DEBUG]: RDS Cluster create response: %s", resp)
 
+	} else if v, ok := d.GetOk("s3_import"); ok {
+		if _, ok := d.GetOk("master_password"); !ok {
+			return fmt.Errorf(`provider.aws: aws_db_instance: %s: "password": required field is not set`, d.Get("name").(string))
```
Comment (bflad) [inline aws/resource_aws_rds_cluster.go]: `"password"` should be `"master_password"` 😄  Same with username one below

Comment (bflad) [reply aws/resource_aws_rds_cluster.go]: We use a special wrapper to parallelize the acceptance testing in our CI environment: https://github.com/jen20/teamcity-go-test/

It'd probably be nice if we offered something similar for local testing.

File: aws/resource_aws_rds_cluster.go [hunk 2]
```diff
@@ -432,6 +471,97 @@ func resourceAwsRDSClusterCreate(d *schema.ResourceData, meta interface{}) error
...
+		err := resource.Retry(5*time.Minute, func() *resource.RetryError {
+			resp, err := conn.RestoreDBClusterFromS3(createOpts)
+			if err != nil {
+				if isAWSErr(err, "InvalidParameterValue", "S3_SNAPSHOT_INGESTION") {
+					return resource.RetryableError(err)
+				}
+				if isAWSErr(err, "InvalidParameterValue", "S3 bucket cannot be found") {
+					return resource.RetryableError(err)
+				}
+				return resource.NonRetryableError(err)
```
[the remark this answers is not in the document]
Comment (jasonmcintosh) [reply aws/resource_aws_rds_cluster.go]: Interesting... can add that, though this SEEMS like an AWS issue.  We had no issues running here... will add this, but dang that seems like a really strange issue.  Possibly a cross region issue?

[the remark this answers is not in the document]
Comment (jasonmcintosh) [reply aws/resource_aws_rds_cluster.go]: Yeah just found it in API docs.  Bucket has to be created in same region as the RDS cluster.

[the remark this answers is not in the document]
Comment (jasonmcintosh) [reply aws/resource_aws_rds_cluster.go]: I'm hoping the latest commit fixes this issue - sets the bucket region explicitly as part of the test.

**Comment by bflad** [comment]
This has been released in [version 1.17.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1170-may-02-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Pull request merged**